.buildkite/pipeline.yml

```diff
@@ -23,34 +23,54 @@ steps:
             # Allow WP-CLI to be run as root, otherwise it throws an exception.
             # Reference: https://git.io/J9q2S
             - "WP_CLI_ALLOW_ROOT=true"
+      - peakon/git-shallow-clone#v0.0.1:
+          depth: 1
```

Comment:
This plugin makes checking out the repo 2 minutes faster

**Before ([build](https://buildkite.com/automattic/gutenberg-mobile/builds/5869#018855c3-bc85-45cb-93be-41386965a05d))**

![image](https://github.com/wordpress-mobile/gutenberg-mobile/assets/1218433/9ba460f9-ab93-41c1-afb4-eeb6f4e39212)


**After ([build](https://buildkite.com/automattic/gutenberg-mobile/builds/5870#0188567c-9da5-4726-85bd-a2b5d132a314))**

<img width="1148" alt="image" src="https://github.com/wordpress-mobile/gutenberg-mobile/assets/1218433/bd751d00-a3e4-44d4-9d03-c4ce9d5909f8">

Reply:
🤔 I guess I could DRY this by defining it in `x-common-params`.

I was hoping there was a way to add a plugin to every step, but [the docs don't mention it](https://buildkite.com/docs/plugins/using#using-yaml-anchors-with-plugins).